portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,91 @@
+getInfo();
+
+function getInfo() {
+    const wrapper = document.getElementById("wrapper");
+    wrapper.innerHTML = `
+        <h1>About Me</h1> 
```

Comment:
I don't really like how the html parts are included inside the JavaScript files. Once you add actual logic to the JavaScript this gets painful to maintain. I do not recommend using this pattern in future projects.

One solution would be to use jQuery's load function: https://api.jquery.com/load/ (I haven't tried this myself but it looks like it allows you to include a html-file instead of text).

Reply:
As suggested, I used jQuery's load function and I got rid of all innerHTML (I also made use of the DOM where there wasn't so much code). I also moved all the functions in one script, as I thought there is no need to have many JS files with few loc. Hope it's better now!